Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py

```diff
@@ -1308,7 +1321,11 @@ def get_modified_remote_data_command(service: client.Service, args):
              '| fields rule_id ' \
              '| dedup rule_id'.format(last_update_splunk_timestamp)
     demisto.debug(f'Performing get-modified-remote-data command with query: {search}')
-    for item in results.ResultsReader(service.jobs.oneshot(search, count=MIRROR_LIMIT)):
+
+    for item in results.JSONResultsReader(service.jobs.oneshot(search, count=MIRROR_LIMIT, output_mode='json')):
+        if isinstance(item, results.Message):
+            demisto.debug(f"Message from splunk-sdk message: {item.message}")
+            continue
```

Comment:
you can define a generic function like:
```
    def handle_messgae(item):
            if isinstance(item, results.Message):
                demisto.debug(f"Message from splunk-sdk message: {item.message}")
                return True
            return False
```
and then use it in all the relevant places 
```
        if handle_message(item):
            continue
```